### PR TITLE
Added link to bug report form in nav bar: #565

### PR DESCRIPTION
**Pull request by tagawa**

### CODE REVIEW

File: webcompat/static/font/webcompat/webcompat.svg [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <svg xmlns="http://www.w3.org/2000/svg">
 <metadata>Generated by IcoMoon</metadata>
 <defs>
-<font id="webcompat" horiz-adv-x="1024">
+<font id="icomoon" horiz-adv-x="1024">
```
Comment (miketaylr) [inline webcompat/static/font/webcompat/webcompat.svg]: Everything else looks good. Any reason for this `webcompat` -> `iconmoon` change? Maybe @magsout knows.

**Comment by magsout** [comment]
this is the configuration in icomoon. You used what conf @tagawa ?

**Comment by magsout** [comment]
you can take icon here https://github.com/webcompat/design/tree/master/3_Style%20Guide/icon%20from%20icomoon (add `selection.json`for configuration)

**Comment by tagawa** [comment]
Thanks for re-doing the fonts @magsout. Looks like I didn't properly.

**Comment by miketaylr** [comment]
Sweet, lgtm. r+

**Comment by tagawa** [comment]
Thanks @miketaylr and @magsout

**Pull request merged**